build.sbt

```diff
@@ -1,5 +1,4 @@
-val scala212Version = "2.12.17"
```

Comment:
no longer support 2.12 anymore?

Reply:
play-guice has support for Scala 2.13 and 3, no longer has updates on Scala 2.12

https://mvnrepository.com/artifact/org.playframework/play-guice
https://mvnrepository.com/search?q=play+guice